### PR TITLE
update contributing.md

### PR DESCRIPTION
**Pull request by jihyeonyi**
### Summary
describe development process more specifically.

### Checklist

- [ ] I have added unit tests to cover my changes.​
- [ ] I have added integration tests to cover my changes.​
- [ ] I have added the description of my changes into [CHANGELOG](https://github.com/openvinotoolkit/datumaro/blob/develop/CHANGELOG.md).​
- [x] I have updated the [documentation](https://github.com/openvinotoolkit/datumaro/tree/develop/docs) accordingly

### License

- [ ] I submit _my code changes_ under the same [MIT License](https://github.com/openvinotoolkit/datumaro/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below).

```python
# Copyright (C) 2023 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

**Comment by jihyeonyi** [comment]
I'm not sure this update is also added to CHANGELOG. If needed, please let me know.

**Review by sooahleex — APPROVED** [review]
Looks good to me. By the way, I think this could be merged into releases/1.4.0.

File: contributing.md [hunk 1]
```diff
@@ -1,41 +1,65 @@
...
+    ```
+
+5. Install test dependencies:
+    ``` bash
+    pip install -r tests/requirements.txt
+    ```
+
+6. Set up pre-commit hooks in the repo. See [Code style](#code-style).
+    ``` bash
+    pre-commit run
```
Comment (vinnamkim) [inline contributing.md:46]: Is it possible without running `pre-commit install` in advance?

Comment (jihyeonyi) [reply contributing.md:46]: Actually.. I've just copied the line from Code style section. Moreover I ran pre-commit install only (without run) and it worked. I'll test this and update soon.

**Review by vinnamkim — APPROVED** [review]

**Pull request merged**